client.go

```diff
@@ -62,14 +64,29 @@ func (c *Client) doReq(req *http.Request) (io.ReadCloser, error) {
 
 	if resp.StatusCode != http.StatusOK {
 		defer resp.Body.Close()
+		retryAfter := getRetryAfter(resp)
 		b := &bytes.Buffer{}
 		io.Copy(b, resp.Body) // Ignore error.
 		return nil, StatusError{
 			Body:       b,
 			Status:     resp.Status,
 			StatusCode: resp.StatusCode,
+			RetryAfter: retryAfter,
 		}
 	}
 
 	return resp.Body, nil
 }
+
+func getRetryAfter(resp *http.Response) int {
+	header := resp.Header.Get("Retry-After")
+	if header != "" {
+		// FullStory always returns the Retry-After header as number
+		// of seconds to wait, so convert it to an int, ignoring errors.
```

Comment:
in the comment, could we remove the "ignoring errors" part, since we do check `err == nil` on the next line?

Reply:
Maybe "swallowing" is a better word than "ignoring"? What I wish to communicate is that the error doesn't bubble up; we simply return 0 if the value is unparsable.

Reply:
How about a doc comment like the following?

> getRetryAfter returns the value of the "Retry-After" header as an integer. If the header isn't present or if there is an error parsing the value, it returns 0.